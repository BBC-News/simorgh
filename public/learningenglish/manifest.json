{
  "name": "BBC Learning English",
  "short_name": "BBC Learning English",
  "Scope": "/learningenglish/",
  "background_color": "#90838B",
  "display": "standalone",
  "theme_color": "#90838B",
  "icons": [
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-72x72.png",
      "sizes": "72x72",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-96x96.png",
      "sizes": "96x96",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-128x128.png",
      "sizes": "128x128",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-144x144.png",
      "sizes": "144x144",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-152x152.png",
      "sizes": "152x152",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-384x384.png",
      "sizes": "384x384",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/learningenglish/images/icons/icon-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "splash_pages": null
}
