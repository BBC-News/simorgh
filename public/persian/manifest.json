{
  "name": "BBC News فارسی",
  "short_name": "BBC News فارسی",
  "Scope": "/persian/",
  "background_color": "#b80000",
  "display": "standalone",
  "start_url": "/persian/articles/cwv2xv848j5o",
  "theme_color": "#b80000",
  "icons": [
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-72x72.png",
      "sizes": "72x72",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-96x96.png",
      "sizes": "96x96",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-128x128.png",
      "sizes": "128x128",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-144x144.png",
      "sizes": "144x144",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-152x152.png",
      "sizes": "152x152",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-384x384.png",
      "sizes": "384x384",
      "type": "image/png"
    },
    {
      "src": "https://news.files.bbci.co.uk/include/articles/public/persian/images/icons/icon-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "splash_pages": null
}
