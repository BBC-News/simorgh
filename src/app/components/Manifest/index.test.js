import manifest from './index';

describe('Manifest', () => {
  test('should render correctly for `news` service', () => {
    process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = 'http://foobar.com';
    expect(manifest('news')).toMatchSnapshot();
  });

  test('should render correctly for `persian` service', () => {
    process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = 'http://foobar.com';
    expect(manifest('persian')).toMatchSnapshot();
  });
});
