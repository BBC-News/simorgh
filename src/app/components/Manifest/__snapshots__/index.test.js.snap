// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manifest should render correctly for \`news\` service 1`] = `
Object {
  "Scope": "/news/",
  "background_color": "#B80000",
  "display": "standalone",
  "icons": Array [
    Object {
      "sizes": "72x72",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "96x96",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "128x128",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "144x144",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "152x152",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "192x192",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "384x384",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "512x512",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
  ],
  "name": "BBC News",
  "short_name": "BBC News",
  "splash_pages": null,
  "start_url": "/news/articles/c9rpqy7pmypo",
  "theme_color": "#B80000",
}
`;

exports[`Manifest should render correctly for \`persian\` service 1`] = `
Object {
  "Scope": "/persian/",
  "background_color": "#B80000",
  "display": "standalone",
  "icons": Array [
    Object {
      "sizes": "72x72",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "96x96",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "128x128",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "144x144",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "152x152",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "192x192",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "384x384",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
    Object {
      "sizes": "512x512",
      "src": "http://foobar.comtest-file-stub",
      "type": "image/png",
    },
  ],
  "name": "BBC News فارسی",
  "short_name": "BBC News فارسی",
  "splash_pages": null,
  "start_url": "/persian/articles/cwv2xv848j5o",
  "theme_color": "#B80000",
}
`;
